selfdrive/ui/ui.cc

```diff
@@ -202,6 +178,57 @@ static void update_sockets(UIState *s) {
   s->started = scene.thermal.getStarted() || scene.frontview;
 }
 
+static void update_alert(UIState *s) {
+  if (!s->started || s->scene.frontview) return;
```

Comment:
why does this check frontview? controlsState shouldn't be updated anyway

Reply:
checking the control times also requires this condition